src/DI/ContainerLoader.php

```diff
@@ -108,7 +108,7 @@ protected function generate($class, $generator)
 	{
 		$compiler = new Compiler;
 		$compiler->getContainerBuilder()->setClassName($class);
-		$code = call_user_func_array($generator, [& $compiler]);
+		$code = $generator(...[& $compiler]);
```

Comment:
Oh I see, this is a nasty hack to workaround call-time pass-by-reference. Imo we should get rid of it anyway :)